docs/reference/operations-guide/core.md

```diff
@@ -0,0 +1,4522 @@
+                .first(new NamedOperation.Builder<EntitySeed, Iterable<? extends Element>>()
+                        .name("1-hop")
+                        .input(new EntitySeed(1))
+                        .build())
+                .then(new Filter.Builder()
+                        .edge("edge", new ElementFilter.Builder()
+                                .select("count")
+                                .execute(new IsMoreThan(2))
+                                .build())
+                        .build())
+                .build();
+        ```
+
+    === "JSON"
+
+        ``` json
+        {
+          "class" : "OperationChain",
+          "operations" : [ {
+            "class" : "NamedOperation",
+            "input" : [ {
+              "class" : "EntitySeed",
+              "class" : "EntitySeed",
+              "vertex" : 1
+            } ],
+            "operationName" : "1-hop"
+          }, {
+            "class" : "Filter",
+            "edges" : {
+              "edge" : {
+                "predicates" : [ {
+                  "selection" : [ "count" ],
+                  "predicate" : {
+                    "class" : "IsMoreThan",
+                    "orEqualTo" : false,
+                    "value" : 2
+                  }
+                } ]
+              }
+            }
+          } ]
+        }
+        ```
+
+    === "Python"
+
+        ``` python
+        g.OperationChain( 
+          operations=[ 
+            g.NamedOperation( 
+              input=[ 
+                g.EntitySeed( 
+                  vertex=1 
+                ) 
+              ], 
+              operation_name="1-hop" 
+            ), 
+            g.Filter( 
+              edges=[ 
+                g.ElementFilterDefinition( 
+                  group="edge", 
+                  predicates=[ 
+                    g.PredicateContext( 
+                      selection=[ 
+                        "count" 
+                      ], 
+                      predicate=g.IsMoreThan( 
+                        value=2, 
+                        or_equal_to=False 
+                      ) 
+                    ) 
+                  ] 
+                ) 
+              ] 
+            ) 
+          ] 
+        )
+        ```
+
+    Results:
+
+    === "Java"
+
+        ``` java
+        Edge[source=1,destination=2,directed=true,matchedVertex=SOURCE,group=edge,properties=Properties[count=<java.lang.Integer>3]]
+        ```
+
+    === "JSON"
+
+        ``` json
+        [ {
+          "class" : "uk.gov.gchq.gaffer.data.element.Edge",
+          "group" : "edge",
+          "source" : 1,
+          "destination" : 2,
+          "directed" : true,
+          "matchedVertex" : "SOURCE",
+          "properties" : {
+            "count" : 3
+          }
+        } ]
+        ```
+
+## Limit
+
+Limits the number of items. [Javadoc](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/operation/impl/Limit.html)
```

Comment:
'Limits the number of items.' feels like an incomplete to me. Would 'Limits the number of items returned by a query' be correct?

Reply:
I've changed it to say "elements returned" instead of "items" and added a bit about truncation. This is not strictly the number of items returned, as the query itself is running differently based on this and it is part of the query, rather than a simple post query trimming of results. This also keeps it more consistent with the other operation descriptions.